static/js/app.js

```diff
@@ -18,7 +18,6 @@ eventsData.sort((a, b) => {
 
       return monthIndexB - monthIndexA;
   }
-
   return yearComparison;
 });
```

Comment:
I would avoid having actual logic on the root of the file. You could move this logic into a function like `loadEventData` or similar and call it "onDocumentReady" or any other initial DOM page event.

Reply:
Will review it, thank you.